src/AssemblerHttpServer.cpp

```diff
@@ -226,7 +198,7 @@ void Assembler::writeHtmlBegin(ostream& html, bool navigation) const
         "<!DOCTYPE html>"
         "<html>"
         "<head>"
-        "<link rel=icon href=docs/CZI-new-logo.png />"
+        "<link rel=icon href=https://chanzuckerberg.github.io/shasta/CZI-new-logo.png />"
```

Comment:
I don't want to add a reliance on GitHub Pages, for various reasons that we can talk about. I would completely remove the link, perhaps leaving it in, commented out, to make it easier to put it back if we decide to do that.

Reply:
We can always put the CZI logo in an s3 bucket and link to it. I will just comment it out for now.